woocommerce/woocommerce
# [Product Block Editor]: propagate content from description inner blocks to Modal editor

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



This PR propagates the changes that the user makes in the Description block to the Modal Editor.



Closes https://github.com/woocommerce/woocommerce/issues/39622



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the Block editor
2. Identify the Description block
3. Confirm the content defined in the Description block is properly propagated to the Modal Editor.


https://github.com/woocommerce/woocommerce/assets/77539/f2d7f4d3-85ed-490b-9df1-6c1733184dcd




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
